### PR TITLE
Middleman support

### PR DESCRIPTION
**Pull request by pjkelly**
This PR adds support for [Middleman](https://middlemanapp.com/) sites.  The other commit in this PR removes a reference to a file that doesn't exist in version control.  With that line left in place in the gemspec, the gem won't build from source.  I'm happy to remove that commit, if I'm misunderstanding what's going on, but thought I'd add for the time-being.

### CODE REVIEW

File: s3_website.gemspec [hunk 1]
```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |s|
   s.files         = `git ls-files`
                       .split("\n")
                       .reject { |f| f.match('sbt-launch.jar') } # Reject the SBT jar, as it is a big file
-                      .push('resources/s3_website.jar.md5') # Include the checksum file in the gem
```
Comment (laurilehmijoki) [inline s3_website.gemspec]: Thanks for the pull request, @pjkelly! One question: why did you remove this line?

Comment (pjkelly) [inline s3_website.gemspec]: I found that when I was building the gem from source (in my fork), the installation was failing because that file doesn't actually exist in the repository.  I can definitely remove the commit that does this if you'd prefer.  Let me know and I'll get it taken care of.

Comment (pjkelly) [inline s3_website.gemspec]: Also, looks like I need to rebase my branch against master - will do that shortly.

**Comment by pjkelly** [comment]
Ok, my branch is up-to-date with master now.  Let me know about the change to the gemspec when you have a chance.

File: s3_website.gemspec (hunk 1 above)
Comment (laurilehmijoki) [inline s3_website.gemspec]: Thanks for the explanation, @pjkelly. I have never ran into that problem because I've always had the *s3_website.jar.md* there. Clearly this is not the case for devs like you who fork the repo and start from scratch.

**Comment by laurilehmijoki** [comment]
@pjkelly thanks for your contribution, I've just released this PR in the version **2.12.1**. I did have to revert the MD5 removal, because the `s3_website` Ruby binary uses that: https://github.com/laurilehmijoki/s3_website/blob/master/bin/s3_website#L207.

**Comment by pjkelly** [comment]
Thanks @laurilehmijoki !
